### ISSUE TITLE
[BUG] Wrong alt-tabbing order when not in window effect mode.

### ISSUE THREAD

**Issue by johnbonda** [linuxdeepin/dde-control-center#114]
ALT-TAB does not work the way it is supposed to when not in Window Effect mode. 
The feature is implemented correctly in Window effect mode, but the alt-tab order doesn't work the same way when Window effect is off.
I'm guessing the problem is when Window effect mode is turned off, ALT-TAB directly switches to the next window instead of previewing it (like in Window effect mode).
In Window effect mode we preview and when we stop ALT-TABBING it switches to that particular window. But since non window effect mode is directly switching windows during ALT-TABBING to the appropriate window, the last opened window is lost. 
Please fix it.

**Comment by bocekm** [comment]
I had the same problem and I reported it here: https://github.com/linuxdeepin/deepin-wm/issues/66
But it's been resolved when upgraded to Fedora 31 (deepin-desktop-5.0.0-3).
